homeassistant/components/knx/light.py

```diff
@@ -180,9 +180,10 @@ def should_poll(self):
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
-        if not self.device.supports_brightness:
-            return None
-        return self.device.current_brightness
+        if self.device.supports_brightness:
+            return self.device.current_brightness
+        if self.device.supports_color and self._hsv_color:
+            return round(self._hsv_color[-1] / 100 * 255)
 
```

Comment:
aren't you missing `return None` now?

Reply:
True, but it will implicitly `return None`, right?
If I add it, won't it trigger the `useless-return` message in pylint?

> Useless return at end of function or method Emitted when a single "return" or "return None" statement is found at the end of function or method definition. This statement can safely be removed because Python will implicitly return None

Reply:
I'm not really a python dev, but I doubt that no return value in case both conditions for some reason might not trigger is the same as returning None.

Reply:
@frenck, can you advise if the explicit `return None` is necessary? AFAIK pylint complains on this, right?

Reply:
All function exits should return something explicitly if the function returns something explicitly anywhere.